backend/main.go

```diff
@@ -129,6 +132,29 @@ func createComment(w http.ResponseWriter, r *http.Request, db *sql.DB){
 
 	responseJSON(w, http.StatusCreated, "Comment created successfully")
 }
+
+//ユーザー追加ハンドラ
+func createUser(w http.ResponseWriter, r *http.Request, db *sql.DB){
+	var user User
+	if err := decodeBody(r, &user); err != nil{
+		responseJSON(w, http.StatusBadRequest, err.Error())
+		return
+	}
+
+	hash, err := bcrypt.GenerateFromPassword([]byte(user.PwHash), bcrypt.DefaultCost)
+	if err != nil {
+		
+	}
+
+	_, err := db.Exec(addUser, user.Name, hash)
```

Comment:
:=だとエラーが出るはずです。（:=は左に新しい変数がある場合のみ）
なので、＝にすると良いと思います
```suggestion
	_, err = db.Exec(addUser, user.Name, hash)
```